server/lib/queries.js

```diff
@@ -20,17 +20,10 @@ const generateFXConversionSQL = aggregate => {
     amountColumn = 'SUM("t."netAmountInCollectiveCurrency"")';
   }
 
-  const fxConversion = [];
-  for (const currency in currencies) {
-    fxConversion.push([currency, currencies[currency].fxrate]);
-  }
-
-  let sql = 'CASE ';
-  sql += fxConversion
-    .map(currency => `WHEN ${currencyColumn} = '${currency[0]}' THEN ${amountColumn} / ${currency[1]}`)
-    .join('\n');
-  sql += 'ELSE 0 END';
-
+  let sql = '';
+  convertToCurrency(1, 'USD', currencyColumn).then(
+    amount => (sql += currencyColumn ? `${amountColumn} / ${amount}` : '0'),
```

Comment:
Because this call is asynchronous, the function will always return an empty string.

Reply:
Understood. Couldn't properly implement this.  Will Fix